salt/modules/mount.py

```diff
@@ -5,6 +5,7 @@
 
 # Import python libs
 from __future__ import absolute_import, print_function, unicode_literals
+from collections import OrderedDict
```

Comment:
We generally prefer the use of `salt.utils.odict` instead.

Reply:
Will do, hang over from developing it.